client/src/main/java/org/apache/cloudstack/ServerDaemon.java

```diff
@@ -116,7 +116,7 @@ public void init(final DaemonContext context) {
         LOG.info("Server configuration file found: " + confFile.getAbsolutePath());
 
         try {
-            final Properties properties = PropertiesUtil.loadFromFile(confFile);
+            final Properties properties = ServerProperties.getServerProperties();
```

Comment:
@Pearl1594 `confFile` object here is only used for getting properties from "_server.properties_".  Can you remove / move the lines above ( L#109 - L#116) to ServerProperties class as applicable.